special-interest-groups/sig-template/roles-and-organization-management.md

```diff
@@ -0,0 +1,73 @@
+# Template SIG Roles and Organization Management
+
+<!--
+Outline the organism specific to this SIG, as well as those that differ from [SIG Governance](sig-governance.md)
+-->
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
```

Comment:
- Suggest combing these two lines.
- `It` looks weird here. If it refers to "instructions", the grammar is incorrect; if it refers to "TiDB Developer Group", it sounds duplicate because there's another "developer group" in the end.